eos/models/wet.cc

```diff
@@ -471,10 +471,16 @@ namespace eos
 
     /* sbnunu Wilson coefficients */
     WilsonScanComponent<components::WET::SBNuNu>::WilsonScanComponent(const Parameters & p, const Options &, ParameterUser & u) :
-    _re_cl(p["sbnunu::Re{cL}"], u),
-    _im_cl(p["sbnunu::Im{cL}"], u),
-    _re_cr(p["sbnunu::Re{cR}"], u),
-    _im_cr(p["sbnunu::Im{cR}"], u)
+    _re_cvl(p["sbnunu::Re{cVL}"], u),
```

Comment:
For some reason the coding style of the original was incorrect. Can you please indent these lines by one level to the right?

Reply:
It is the same for `WilsonScanComponent<components::WET::CBLNu>::WilsonScanComponent`, I will change it there too.